packages/victory-core/src/index.d.ts

```diff
@@ -540,6 +540,7 @@ export interface VictoryCommonProps {
   minDomain?: number | { x?: number; y?: number };
   name?: string;
   padding?: PaddingProps;
+  polar?: boolean;
```

Comment:
I can see that you've removed `polar` from a few of the interfaces that already extend `VictoryCommonProps` to reduce redundancy, but would you mind removing it in another spot? I see it in `VictoryCandleStickProps` at first glance, but there might be more duplicity elsewhere.

Reply:
(I searched for `polar?` in the codebase to pull up duplicate instances 🙂)

Reply:
Yep, moved in in https://github.com/FormidableLabs/victory/pull/1529. Was just going to merge that first so I don't have surprise conflicts

Reply:
Oh heck, I misinterpreted this. Soz, yes, will audit all of them and pare them down!